IAM/Frequently_asked_questions.mediawiki

```diff
@@ -0,0 +1,49 @@
+=== Mozilla IAM FAQ (Frequently Asked Questions) ===
+
+==== '''Q''': ''What is Mozilla IAM?'' ====
+
+Mozilla IAM stands for Mozilla's Identity and Access Management. It's the internal login system that Mozilla uses to login to various web properties and systems.
```

Comment:
how about: "It's the internal systems that Mozilla uses to manage logins to various"

Reply:
Do we need the term _internal_ at all? Would "It's the system that Mozilla uses to manage ..." also work?

Reply:
Do we want to link the initial "Mozilla IAM" to this site: https://github.com/mozilla-iam/mozilla-iam ?

Reply:
internal: yes, or something similar to indicate that this is NOT firefox account or personna. We could also just say "this is not firefox accounts or personna"

Reply:
link done